tools/merge_note_registries.py

```
@@ -0,0 +1,137 @@
+
+    return additions
+
+
+def _load_json(path: Path) -> dict:
+    if not path.exists():
+        logging.warning(f"Registry not found, treating as empty: {path}")
+        return {}
+    try:
+        with open(path, "r", encoding="utf-8") as f:
+            return json.load(f)
+    except (json.JSONDecodeError, IOError) as e:
```

Ruff (0.16.1)

[warning] Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)